components/src/dynamo/vllm/tests/test_vllm_worker_factory.py

```
@@ -718,21 +718,25 @@ async def _noop(*_args, **_kwargs) -> None:
 ) -> None:
     engine_client = Mock()
-    vllm_config = Mock(additional_config={})
+    vllm_config = Mock(
+        additional_config={},
+        cache_config=SimpleNamespace(num_gpu_blocks=96),
+    )
     engine_tuple: EngineSetupResult = (
         engine_client,
         vllm_config,
         Mock(),
         "/tmp/prom",
```

Ruff (0.16.0)

[error] Probable insecure usage of temporary file or directory: "/tmp/prom"

(S108)